Game/Source/GDKTestGyms/BenchmarkGymGameModeBase.cpp

```diff
@@ -414,4 +510,25 @@ void ABenchmarkGymGameModeBase::SetTotalNPCs(int32 Value)
 void ABenchmarkGymGameModeBase::OnRepTotalNPCs()
 {
 	OnTotalNPCsUpdated(TotalNPCs);
+}
+
+int32 ABenchmarkGymGameModeBase::GetActorClassCount(TSubclassOf<AActor> ActorClass) const
+{
+	const UCounterComponent* CounterComponent = Cast<UCounterComponent>(GameState->GetComponentByClass(UCounterComponent::StaticClass()));
+	check(CounterComponent != nullptr)
+
+	return CounterComponent->GetActorClassCount(ActorClass);
+}
+
+void ABenchmarkGymGameModeBase::SetLifetime(int32 Lifetime)
+{
+	if (TestLifetimeTimer.SetTimer(Lifetime))
```

Comment:
Was this getting called multiple times? What's the value to not allowing it to be reset?

Reply:
Yes it get's called multiple times but actually... maybe the solution is to not call it twice. I have a feeling it "needs" to be called twice but I will check that.

Reply:
Gonna keep this code for now. Ran a tests trying to fix this but it failed and I don't really want to spend time looking into it right now. I think this code is acceptable for now but I will create a follow up ticket to work out why we call the parseparameters a second time. Let me know if that is good with you.